### PR TITLE
Adds a clinic's city & state to the clinic dropdown

### LINKED ISSUE #2188: Add city/state to the clinics dropdown in patient edit view
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**

If you go to a patient edit page, here's how the clinic dropdown renders:

image

Those are just the names of the clinics, and they're a little tough to read on the fly - where is `Sample Clinic with NAF`? Since geography matters quite a bit, I think it would be really nice to add city/state to the dropdown. So, that screenshot should change to something like this:

```
Sample Clinic 1 - DC (Washington, DC)
Sample Clinic 2 - VA (Arlington, VA)
Sample Clinic with NAF (Washington, DC)
```
etc. 

* **What feature or behavior is this required for?**

A nice quality of life improvement for CMs.

* **How could we solve this issue? (Not knowing is okay!)**

Take a look at `clinic_options` in `patients_helper.rb`. You might have to muck with i18n a little bit, but changing around the clinic name a bit is probably the way to go.

* **Anything else?**
Note that `Sample Clinic 1 - DC` and `Sample Clinic 2 - VA` are the names of the clinic in the seedfile - we definitely are not doing this right now!

### PR DESCRIPTION
**Pull request by nsiwnf**
I rule and have completed some work on Case Manager that's ready for review!

This pull request makes the following changes:
* Adds clinic city, state to the drop-down in "Patient/edit" > "Abortion Information"

(If there are changes to the views, please include a screenshot so we know what to look for!)

It relates to the following issue #s: 
* Resolves #2188

### CODE REVIEW

**Comment by lomky** [comment]
before and after of clinic dropdown

**Review by lomky — APPROVED** [review]
Wonderful work! Thank you! 🎉 🥇

File: app/helpers/patients_helper.rb [hunk 1]
```diff
@@ -112,7 +112,7 @@ def clinic_options
     clinics = Clinic.all.sort_by(&:name)
     active_clinics = clinics.select(&:active)
                             .map { |clinic| [
-                              clinic.name,
+                              t('patient.abortion_information.clinic_section.clinic_display', clinic_name: clinic.name, city: clinic.city, state: clinic.state),
```
Comment (colinxfleming) [inline app/helpers/patients_helper.rb:115]: love your inclination to i18n this 💯

**Comment by lomky** [comment]
@all-contributors please add @nsiwnf  for code!

**Pull request merged**